eureka-webapp/src/main/java/edu/emory/cci/aiw/cvrg/eureka/servlet/UserAcctManagerServlet.java

```diff
@@ -84,8 +85,15 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
 		ServletWorker worker;
 
 		if (action != null && action.equals("save")) {
-			LOGGER.info("Saving user");
-			worker = new SaveUserAcctWorker(this.getServletContext(), this.servicesClient);
+			String id = req.getParameter("id");  
+			if(id!=null && !id.isEmpty()){   
+				LOGGER.info("Saving user info");
+				worker = new SaveUserAcctInfoWorker(this.getServletContext(), this.servicesClient);                             
+			} else {
```

Comment:
I believe a better approach would be to add a new action for a password save request, like savepassword, rather than relying on the id parameter being there or not.